ACE/ace/Malloc_T.cpp

```diff
@@ -177,6 +177,171 @@ ACE_Dynamic_Cached_Allocator<ACE_LOCK>::free (void * ptr)
     this->free_list_.add ((ACE_Cached_Mem_Pool_Node<char> *) ptr);
 }
 
+template <class ACE_LOCK>
+ACE_Cascaded_Dynamic_Cached_Allocator<ACE_LOCK>::ACE_Cascaded_Dynamic_Cached_Allocator
+  (size_t initial_n_chunks, size_t chunk_size)
+    : initial_n_chunks_ (initial_n_chunks),
+      chunk_size_ (chunk_size)
+{
+  ACE_ASSERT (chunk_size_ > 0);
+
+  comb_alloc_ptr tmp;
```

Comment:
initialize tmp

Reply:
the temp var will be assigned or call flow is ended by `ACE_NEW` macro

Should it be initialized?

Reply:
the temp var will be assigned or call flow is ended by `ACE_NEW` macro

Should it be initialized?